src/lustre/lustreAstNormalizer.mli

```diff
@@ -84,7 +84,7 @@ val normalize : TypeCheckerContext.tc_context ->
   LustreAst.t ->
     GeneratedIdentifiers.t GeneratedIdentifiers.StringMap.t ->
   (LustreAst.declaration list * GeneratedIdentifiers.t GeneratedIdentifiers.StringMap.t *
-   [> `LustreAstNormalizerWarning of Lib.position * warning_kind ] list, [> error])
+   [> `LustreAstNormalizerWarning of Lib.position * warning_kind] list * TypeCheckerContext.tc_context, [> error])
```

Comment:
Is the context really modified?

Reply:
addressed-- the context is not modified and this is a change that should have been reverted when desugaring of choose op was moved to a separate file